ui/src/shared/components/Dygraph.js

```diff
@@ -10,6 +10,12 @@ import getRange from 'shared/parsing/getRangeForDygraph'
 import {LINE_COLORS, multiColumnBarPlotter} from 'src/shared/graphs/helpers'
 import DygraphLegend from 'src/shared/components/DygraphLegend'
 
+const hasherino = (str, len) =>
```

Comment:
here's the distribution i'm getting with the current hashing function when i create a dashboard that contains a line for every field under the default `telegraf.autogen`
```
0: 3
1: 2
2: 4
3: 2
4: 3
5: 3
6: 1
7: 4
8: 6
9: 4
10: 3
11: 2
12: 4
```
--> somewhat even, with particular outliers at `6` and `8`. i wonder if there's some different hyperparameters adjustments that could be done, or a slightly different hashing scheme that, could result in a more consistently even distribution, so that all of the colors show up at generally the same frequency?

for that matter, this seems like a good opportunity to write a test for the hashing function, included in our spec, to ensure that it's always creating an even distribution and hitting every value in its color options.